canonical/istio-beacon-k8s-operator
# feat: generate `AuthorizationPolicies` based on service-mesh relation data

## Issue
Adds the feature of creating `AuthorizationPolicies` based on the data provided from applications related on the `service-mesh` relation.  

## Testing instructions
WIP: Integration tests still WIP.  See the integration tests which use a tester charm that can be either side of the service mesh relation.  

## Drive-bys

Some changes are made to the service_mesh.py library based on issues hit during implementing this PR.
